File: src/main/java/hu/telekom/tkriszti/flightInfo/service/PilotInfoService.java
package hu.telekom.tkriszti.flightInfo.service;

import hu.telekom.tkriszti.flightInfo.dto.ResultDTO;

import java.sql.SQLException;
import java.util.Set;

interface PilotInfoService {
    Set<ResultDTO> getPilotData(String pilotName) throws SQLException;
}

File: src/main/kotlin/hu/telekom/tkriszti/flightInfo/service/PilotInfoService.kt
package hu.telekom.tkriszti.flightInfo.service

import hu.telekom.tkriszti.flightInfo.dto.ResultDTO
import org.springframework.stereotype.Service
import java.sql.SQLException

interface PilotInfoService {
    @Throws(SQLException::class)
    fun getPilotData(pilotName: String): Set<ResultDTO?>?
}